javatests/arcs/core/host/BUILD

```diff
@@ -40,6 +41,16 @@ arcs_kt_schema(
     ],
 )
 
+arcs_kt_plan(
+    name = "plans",
+    srcs = [
+        "person.arcs",
+    ],
+    deps = [
```

Comment:
Has this had a buildifier pass? I assumed the arrays could be one-liners.

Reply:
That's post buildifier :)